### ISSUE TITLE
Output of HtmlAnalysis cannot be rendered offline

### ISSUE THREAD

**Issue by saudet** [deeplearning4j/DataVec#176]
To visualize the output created by HtmlAnalysis, it would be helpful sometimes to be able to open it in a browser and have something displayed even without having access to the Internet. The culprit is probably these dependencies:
https://github.com/deeplearning4j/DataVec/blob/master/datavec-api/src/resources/templates/analysis.ftl#L104

**Comment by saudet** [comment]
@AlexDBlack I think this was fixed?

**Comment by AlexDBlack** [comment]
No, it's still definitely an issue: https://github.com/deeplearning4j/DataVec/blob/master/datavec-api/src/resources/templates/analysis.ftl#L104-L110

The only way I can see to fix this is to embed those .css and .js files in the analysis.ftl file itself (either copy + paste, or do it in here: https://github.com/deeplearning4j/DataVec/blob/master/datavec-api/src/main/java/org/datavec/api/transform/ui/HtmlAnalysis.java

**Comment by huitseeker** [comment]
@slinlee if you have an idea on how to create an embeddable file ?

**Comment by AlexDBlack** [comment]
Now that I think about it: I'm already embedding a JS file in a HTML using freemarker here: it should just be a case of using the same approach as there...

https://github.com/deeplearning4j/deeplearning4j/blob/master/deeplearning4j-ui-parent/deeplearning4j-ui-components/src/main/java/org/deeplearning4j/ui/standalone/StaticPageUtil.java#L77-L88

**Comment by slinlee** [comment]
Yeah that sounds like a good approach for offline. The downside is you lose
the caching in the clients browser.
